etc/spack/defaults/windows/resource.yaml

```diff
@@ -0,0 +1,8 @@
+bootstrap-resource:
+  # If set to false Spack will not fetch bootstrap resources but instead
+  # proceed without fetching said resources
+  enable: true
```

Comment:
I thought some users might take objection with the inability to prevent the download of semi-arbitrary binaries, particularly if they want to vendor those binaries themselves.
That was 100% of my reasoning, if you don't think it's a relevant concern, I'll pull it out no problem.

Reply:
> if we ought to collect these sorts of concerns into a higher-level dont-download-things variable 

I'm in support of any degree of centralization we can pull from the configs. I agree that it might be out of scope for this MR though.

> could we have a prior check: look for `file` and only download it if it isn't present?

Great idea, will implement.